server/service/gov/kie/kie_distributor.go

```diff
@@ -56,7 +56,7 @@ const (
 	Rules           = "rules"
 )
 
-var PolicyNames = []string{"retry", "rateLimiting", "circuitBreaker", "bulkhead", "instanceIsolation"}
+var PolicyNames = []string{"retry", "rateLimiting", "circuitBreaker", "bulkhead", "instanceIsolation", "faultInjection"}
```

Comment:
next time, if there is any new policy kind, make this as configable, do not make any code change again

Reply:
Goood idea~